### PR TITLE
Disallow credentials from cross-origin reports

### PR DESCRIPTION
**Pull request by clelland**
Closes: #161

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/reporting/pull/186.html" title="Last updated on Oct 15, 2020, 8:56 PM UTC (cb1df43)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/reporting/186/ce47136...cb1df43.html" title="Last updated on Oct 15, 2020, 8:56 PM UTC (cb1df43)">Diff</a>

### CODE REVIEW

**Comment by clelland** [comment]
Ping @johnwilander -- can you take a quick look and see if this is okay?

**Comment by yoavweiss** [comment]
@johnwilander - friendly ping! :)

**Comment by jyasskin** [comment]
Is there a reason this needs to be `same-origin` instead of `omit`? It would be nice for it to be possible at least in theory to have reports send information that's fully disconnected from the user who experienced it.

**Comment by jyasskin** [comment]
Never mind; I hadn't re-read https://github.com/w3c/reporting/issues/161#issuecomment-506738278. @clelland, could you add a note along those lines to the change, so readers can understand the current state better?

**Comment by clelland** [comment]
@jyasskin, does the note I've added now seem like it covers the right points?

**Review by jyasskin — APPROVED** [review]
Thanks!

File: index.src.html [hunk 1]
```diff
@@ -916,11 +916,20 @@ <h3 id="try-delivery" algorithm>
...
       ::  set
       :   `credentials`
-      ::  "`include`"
+      ::  "`same-origin`"
       :   `body`
       ::  A [=/body=] whose [=body/source=] is the [=byte sequence=] resulting
           from executing [=serialize JSON to bytes=] on |collection|.
 
+      Note: Reports are sent with credentials set to `same-origin`. This allows
+      reporting endpoints which are same-origin with the reporting site to get
```
Comment (jyasskin) [inline index.src.html:925]: "Reporting page" would be better, since "sites" are bigger than "origins".

**Pull request merged**